pkg/distributor/distributor.go

```diff
@@ -154,6 +154,9 @@ type Config struct {
 
 	// for testing
 	ingesterClientFactory ring_client.PoolFactory `yaml:"-"`
+
+	// when true the distributor does not validate labels at ingest time
```

Comment:
Would you mind adding something like "Cortex doesn't directly use it (and should never use it) but this feature is used by other projects built on top of it.".

Reply:
Sure, adding that